### DISCUSSION TITLE
How can we retrive Product Reviews from selling Partner API

### DISCUSSION THREAD

**Discussion by IamSantosh** [amzn/selling-partner-api-models#3083] [General]
Hi there,

I'm using Selling Partner API for fetching the User's Product listed on Amazon. I've successfully fetched the Product details and images using 'catalog/2020-12-01/items' API but I didn't find any API using which I can get Reviews published by customers on that Product.

Please help me out here. **How can I find reviews published on Amazon products using API**?

**Comment by sungolivia** [comment]
@IamSantosh hey unfortunately this is not available via SP-aPI at the moment! We have taken this feedback previously as feature request and will be updating this thread once I got more news!

For the rest of folks, if this is a feature you need please reply or react! Thanks!

**Reply by cohlar** [reply]
Definitely a feature I would need

**Reply by Lokeesh-27** [reply]
Is there any SP-API to retrive Amazon seller feedback / Feedback Manager. Did amazon provide any SP-API or can we access it in Buyer-Seller Messages ?

**Reply by joyceeeen** [reply]
(2) Is there any update on this feature? @sungolivia

**Reply by gokigoks** [reply]
Hey, any updates on this feature? @sungolivia

**Reply by cevaboyz** [reply]
Any update?

**Reply by 0xs3n4** [reply]
Any update?

**Reply by RKDISARKADY** [reply]
Any update?

**Reply by titanleaff** [reply]
Any update?

**Reply by mrcharleshu** [reply]
Any update?

**Reply by RoDoerIng** [reply]
Is this feature anywhere on the roadmap?

**Reply by catchex** [reply]
Any update?

**Reply by cgonzalez-neostella** [reply]
Any update?

**Reply by catchex** [reply]
And I want to get data of bought in past month yet

**Reply by danysz** [reply]
+1

**Reply by liuzhenggang** [reply]
+1

**Reply by kaleabtesfaye** [reply]
+1. Any updates?

**Reply by mkblitz** [reply]
+1 would like this feature. Please update

**Reply by happy-bear-x** [reply]
+1

**Reply by itisdesign** [reply]
+1 for ability to retrieve verified feedback w/ its corresponding orderItemID

**Reply by SylvieManx** [reply]
+1 In fact, from the sellers' point of view, they need these reviews because they want to see the advantages and disadvantages of their products and improve them through review analysis. However, Amazon API does not have such an interface. Many developers have to make review crawlers to meet the needs of sellers. I believe they know the traffic cost of crawlers and the interfaces provided by Amazon itself.

**Comment by harlowh** [comment]
Regarding [B2B data building](https://www.globaldatabase.com/why-use-b2b-data-providers-apis-when-building-your-app), if you're delving into the realm of customer reviews programmatically, it's essential to note that the process often involves leveraging the Amazon Customer Reviews (ACR) API. This API operates independently from SP-API and enables you to systematically fetch customer reviews for a specific product on Amazon. It's crucial to be aware that gaining access to the ACR API requires approval from Amazon, necessitating an application for access. This approach ensures the integrity of the data retrieval process, especially when engaged in B2B data building. Feel free to explore this avenue for a comprehensive understanding of customer sentiments!

**Reply by Lokeesh-27** [reply]
do we need to raise approval in api-services-support@amazon.com ? will ACR API will provide seller feedback also ? @harlowh

**Reply by sidgate** [reply]
Where can I read more about Amazon Customer Reviews (ACR) API ? I couldn't find any documentation?

**Comment by rugved-bodke-stigya** [comment]
It's been 2 years, still we are getting nothing!

**Comment by SylvieManx** [comment]
In my opinion, Amazon does not provide this interface, forcing many developers to crawl reviews through crawlers.
